test/blackbox-tests/test-cases/install/install-glob/install-glob-relative.t

```diff
@@ -0,0 +1,33 @@
+Install a glob pattern from a relative path
+
+  $ cat >dune-project <<EOF
+  > (lang dune 3.6)
+  > (package (name foo))
+  > EOF
+
+  $ mkdir -p stanza stuff/xy
+  $ touch stuff/foo.txt stuff/xy/bar.txt
+
+  $ cat >stanza/dune <<EOF
+  > (install
+  >  (files (glob_files_rec ../stuff/*.txt))
+  >  (section share))
+  > EOF
+
+  $ dune build foo.install
+  $ cat _build/default/foo.install
+  lib: [
+    "_build/install/default/lib/foo/META"
+    "_build/install/default/lib/foo/dune-package"
+  ]
+  share: [
+    "_build/install/default/share/stuff/foo.txt" {"../stuff/foo.txt"}
+    "_build/install/default/share/stuff/xy/bar.txt" {"../stuff/xy/bar.txt"}
+  ]
```

Comment:
It took me a minute to work out why this isn't ok. I assume that all files installed into `share` by the package `foo` must be under `.../share/foo`. Can you add a note explaining this here.